lib/testing-helpcenter/articles_with_tag_popup.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
Since this file is part of `Tags` page it's better to locate it in

```
lib/testing-helpcenter/help_center_tags/articles_with_tag_popup`
```

So it will be a subfolder with same name as main file `help_center_tags`

And change `require` in `help_center_tags` accordingly